assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -69,13 +69,13 @@ function DropTargetsProvider({ children }) {
     });
   }, []);
 
-  const isDropSource = (type) => {
+  const isDropSource = useCallback((type) => {
```

Comment:
These should just be moved as a top-level functions. They are completely static.

This might be lintable. /cc @swissspidy do you know of any linter that could catch all fully static functions and suggest they be moved to the module level?

Reply:
Not off the top of my head.